src/components/Pickups/PickupEdit.vue

```diff
@@ -21,7 +21,7 @@
         :error="hasError('date')"
         :error-label="firstError('date')"
         >
-        <q-datetime type="date" v-model="edit.date" :display-value="$d(edit.date, 'dateShort')"/>
+        <q-datetime type="date" v-model="edit.date" :min="this.today" :display-value="$d(edit.date, 'dateShort')"/>
```

Comment:
You can leave out the `this.` in templates

Reply:
Oops! I knew that. Sorry.

Reply:
Fix pushed